dio/lib/src/adapters/browser_adapter.dart

```diff
@@ -2,6 +2,7 @@ import 'dart:async';
 import 'dart:convert';
 import 'dart:html';
 import 'dart:typed_data';
+import 'package:meta/meta.dart';
```

Comment:
`meta` should be imported by the pubspec since we explicitly use it.

```suggestion

import 'package:meta/meta.dart';
```

Reply:
> `meta` should be imported by the pubspec since we explicitly use it.

where should it be placed in? dependencies or dev_dependencies?

Reply:
In dependencies.

Reply:
> In dependencies.

which version should be imported? I imported the version 1.9.0 and caused some problems.